### DISCUSSION TITLE
Actividad 0 - Baez, Gonzalo

### DISCUSSION THREAD

**Discussion by GonzaloEBaez** [frlp-utn-ingsoft/catedra#62] [Show and tell]
https://github.com/GonzaloEBaez/Actividad-0---Introduccion-a-GIT-Baez-Gonzalo.git

**Comment by RodrigoJacznik** [comment]
Hola, el repo esta vacío. Solo tiene el README.md.

**Reply by GonzaloEBaez** [reply]
Buen dia Rodrigo. En la rama master tambien??

**Reply by RodrigoJacznik** [reply]
En la rama master hay código, tenes razon. De todas formas, no pude correr la app. Vos pudiste?
Hay varios errores, el tutorial va evolucionando el código, cada paso se construye sobre el anterior. Por ejemplo: en la parte 2 se crea estos modelos
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/2715743/5f4caa51-8372-4546-a733-83eafb1bc5f3)
Mas adelante, a esos modelos se le agrega el metodo `__str__`
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/2715743/385e5956-0205-4eca-bb18-97ce68411875)
El comentario `#...` significa que mantengas el código anterior.

Te recomiendo que empieces de nuevo en la rama main y vayas haciendo el tutorial con la app funcionando y cuando algo te genere un error no continues e intentes fixear ese error (podes subir un Q&A)

La app tiene que funcionar para aprobar la actividad.

**Reply by GonzaloEBaez** [reply]
Buenas tardes Rodri, ahí corregí muchas cosas. 

De igual forma tengo algunas dudas que me gustaria consultarte en clases antes de comenzar la clase, voy a hacer lo posible por estar temprano en la facultad.

**Comment by GonzaloEBaez** [comment]
Perfecto, eso era lo que no entendía.  

Ahora lo corrijo antes de la clase. Mil gracias!!

**Reply by RodrigoJacznik** [reply]
La actividad esta aprobada 👍
